### ISSUE TITLE
CPU test failed with betrayal in Antara

### ISSUE THREAD

**Issue by AITUS95** [dosbox-staging/dosbox-staging#3561]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### Different version than latest?

0.81.0 GIT

### What Operating System are you using?

None

### Relevant hardware info

I5 9600K

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

The same issue popped up in this [dosbox-x commit](https://github.com/joncampbell123/dosbox-x/issues/4731), but it got sorted out without knowing what caused it.

![294277047-6ee8e4a3-b8a8-4184-a0ff-c0c589fbe649](https://github.com/dosbox-staging/dosbox-staging/assets/48457684/80acae27-2428-45c8-a50a-88b6ddc224f2)

![294279514-eac35a79-1872-43e0-8614-ebffc10ad493](https://github.com/dosbox-staging/dosbox-staging/assets/48457684/da89ff73-a943-4496-8df5-72b898df65e1)

### Steps to reproduce the behaviour.

1. Start Windows 3.11 in a mounted folder using DOSBox-X with the Windows 3.11.conf configuration file.
2. Begin the installation of Betrayal in Antara via disk.
3. Go through the steps to perform the tests.
4. The installation returns an error in the CPU test."

### Your configuration

```ini
https://gist.github.com/AITUS95/d2b26def53eab6c8463133a92119a632
```

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by johnnovak** [comment]
@AITUS95 Does this work with the same config in 0.80.1?

Does setting `cpu` to `pentium_slow` fix it?

**Comment by johnnovak** [comment]
Also, looking at the dialog it seems this is just a warning that you can ignore. So you can click "Yes" to proceed with the installation and play the game anyway.

Can you confirm that too please @AITUS95?

**Comment by AITUS95** [comment]
@johnnovak I tried setting cputype to 'pentium_slow' but no luck, even dosbox version 0.80.1 doesn't help. You can safely skip the test by pressing 'no', so you can play without issues. But this message shouldn't appear on real hardware.

**Comment by johnnovak** [comment]
> @johnnovak I tried setting `cputype` to `pentium_slow `but no luck, even dosbox version 0.80.1 doesn't help. You can safely skip the test by pressing 'no', so you can play without issues. But this message shouldn't appear on real hardware.

Thanks for confirming it's not a regression.

While it would be nicer if the message did not appear, it's hardly a problem, so this is a very low priority issue.

If you ever find out what fixed this in DOSBox-X, please let us know.

**Comment by AITUS95** [comment]
Yeah, I actually like the idea of DOSBox behaving like a real PC. I'll check out which build of DOSBox-X fixed this issue.

**Comment by johnnovak** [comment]
> Yeah, I actually like the idea of DOSBox behaving like a real PC. I'll check out which build of DOSBox-X fixed this issue.

But it doesn't and never will—it's a non-goal for the project. The goal is running games and programs as well on a PC, but emulating a PC _itself_ accurately is a non-goal.

Might seem contradictory, but it's actually not 😄 

That's PCem/86box territory. See our about page about this:
https://dosbox-staging.github.io/about/

**Comment by Grounded0** [comment]
Dropping https://github.com/dosbox-staging/dosbox-staging/labels/game%20compatibility since the game itself runs properly.
